README.md

```diff
@@ -1 +1,28 @@
-# EmployMe
\ No newline at end of file
+# EmployMe
```

Comment:
No need to mention `Line 3`, `# How would I like to improve the world?`, `Line 11`, `? in Line 17`